x-pack/plugins/canvas/shareable_runtime/webpack.config.js

```diff
@@ -33,7 +33,6 @@ module.exports = {
   // Include a require alias for legacy UI code and styles
   resolve: {
     alias: {
-      ui: path.resolve(KIBANA_ROOT, 'src/legacy/ui/public'),
```

Comment:
I suspect we can clean up other aliases in the file as well. @elastic/kibana-presentation